src/terminal/parser/OutputStateMachineEngine.hpp

```diff
@@ -131,8 +131,13 @@ namespace Microsoft::Console::VirtualTerminal
             SetIconAndWindowTitle = 0,
             SetWindowIcon = 1,
             SetWindowTitle = 2,
+            SetWindowProperty = 3, // Not implemented
             SetColor = 4,
+            SetForegroundColor = 10, // WIP

```

Comment:
These two WIP comments seem out of date now :P

Reply:
Oops. You're right. I totally forgot to remove those. They were there for the sake of a livestreaming session 😊